patroni/postgresql/mpp/citus.py

```diff
@@ -408,8 +411,7 @@ def adjust_postgres_gucs(self, parameters: Dict[str, Any]) -> None:
         parameters['citus.local_hostname'] = self._postgresql.connection_pool.conn_kwargs.get('host', 'localhost')
 
     def ignore_replication_slot(self, slot: Dict[str, str]) -> bool:
```

Comment:
```suggestion
    def ignore_replication_slot(self, slot: Dict[str, str]) -> bool:
        """Check whether provided replication *slot* existing in the database should not be removed.
        
        .. note::
            MPP database may create replication slots for its own use, for example to migrate data between workers
            using logical replication, and we don't want to suddenly drop them.
            
        :param slot: dictionary containing the replication slot settings, like ``name``, ``database``, ``type``, and
                     ``plugin``.
                     
        :returns: ``True`` if the replication slots should not be removed, otherwise ``False``.
        """
```